sync.py

```diff
@@ -122,7 +121,9 @@ def parallel_sync(self):
                     logger.propagate = 0
 
                     self.glancesync.sync_region(region)
-                    sys.exit(0)
+                    print pid
```

Comment:
It's a forgotten trace. By the way, I always will use print instead of print() in traces, because you have a radar to detect them :-D